### PR TITLE
Add information about next/prev tab keybindings

### PR DESCRIPTION
**Pull request by valdemarua**

### CODE REVIEW

File: docs/keybindings/Keybindings_nl.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
   <kbd>R</kbd>: verversen
   <kbd>+</kbd>: next screen mode (normal/half/fullscreen)
   <kbd>_</kbd>: prev screen mode
+  <kbd>]</kbd>: next tab
```
Comment (mjarkk) [inline docs/keybindings/Keybindings_nl.md]:
```suggestion
  <kbd>]</kbd>: volgende tab
```

File: docs/keybindings/Keybindings_nl.md [hunk 2]
```diff
@@ -12,6 +12,8 @@
...
+  <kbd>[</kbd>: prev tab
```
Comment (mjarkk) [inline docs/keybindings/Keybindings_nl.md]:
```suggestion
  <kbd>[</kbd>: vorige tab
```

**Comment by jesseduffield** [comment]
I should probably work on some contributing guidelines for this, but the keybindings files are actually generated automatically in `scripts/generate_cheatsheet.go`. If a keybinding doesn't appear in the output from that script, it's because it doesn't have a Description against the keybinding in `pkg/gui/keybindings.go`. I would make the change there, and put the relevant translation in `pkg/i18n/english.go` (and other translation files that you have translations for).

Let me know if you need any pointers :)

**Comment by valdemarua** [comment]
@jesseduffield that totally makes sense! I saw that `.go` with locales but I did not get that doc files generated from them. Did not work with `Go` much )

I will do that. 
Thank you for the feedback.

**Comment by jesseduffield** [comment]
@valdemarua how is this coming along?

**Comment by jesseduffield** [comment]
Given that a few people are raising issues about this, I'm gonna go ahead and close this in favour of https://github.com/jesseduffield/lazygit/pull/732 but don't let this dissuade you from making future contributions to lazygit!

**Pull request closed without merging**